2024-03-04 13:03:16,916 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-04 13:03:16,918 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Configure stats pid to 22592
2024-03-04 13:03:16,918 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-04 13:03:16,919 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2\wandb\settings
2024-03-04 13:03:16,919 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-04 13:03:16,919 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-04 13:03:16,920 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-04 13:03:16,922 INFO    MainThread:22592 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2\wandb\run-20240304_130316-dovv3h1w\logs\debug.log
2024-03-04 13:03:16,924 INFO    MainThread:22592 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2\wandb\run-20240304_130316-dovv3h1w\logs\debug-internal.log
2024-03-04 13:03:16,924 INFO    MainThread:22592 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000273549B5400>
2024-03-04 13:03:16,924 INFO    MainThread:22592 [wandb_init.py:init():566] calling init triggers
2024-03-04 13:03:16,924 INFO    MainThread:22592 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-04 13:03:16,925 INFO    MainThread:22592 [wandb_init.py:init():591] re-initializing run, found existing run on stack: d5a5bx32
2024-03-04 13:03:16,926 INFO    MainThread:22592 [wandb_run.py:_finish():1970] finishing run yanzhuo/attention_classifier/d5a5bx32
2024-03-04 13:03:16,927 INFO    MainThread:22592 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-04 13:03:16,929 INFO    MainThread:22592 [wandb_run.py:_restore():2193] restore
2024-03-04 13:03:16,930 INFO    MainThread:22592 [wandb_run.py:_restore():2199] restore done
2024-03-04 13:03:23,821 INFO    MainThread:22592 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-04 13:03:23,821 INFO    MainThread:22592 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-04 13:03:23,851 INFO    MainThread:22592 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-04 13:03:23,903 INFO    MainThread:22592 [wandb_init.py:init():616] starting backend
2024-03-04 13:03:23,903 INFO    MainThread:22592 [wandb_init.py:init():620] setting up manager
2024-03-04 13:03:23,946 INFO    MainThread:22592 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-04 13:03:23,947 INFO    MainThread:22592 [wandb_init.py:init():628] backend started and connected
2024-03-04 13:03:24,034 INFO    MainThread:22592 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-04 13:03:24,040 INFO    MainThread:22592 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 3796: character maps to <undefined>
2024-03-04 13:03:24,040 INFO    MainThread:22592 [wandb_init.py:init():720] updated telemetry
2024-03-04 13:03:24,175 INFO    MainThread:22592 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-04 13:03:25,472 INFO    MainThread:22592 [wandb_run.py:_on_init():2262] communicating current version
2024-03-04 13:03:25,911 INFO    MainThread:22592 [wandb_run.py:_on_init():2271] got version response 
2024-03-04 13:03:25,912 INFO    MainThread:22592 [wandb_init.py:init():804] starting run threads in backend
2024-03-04 13:03:26,201 INFO    MainThread:22592 [wandb_run.py:_console_start():2241] atexit reg
2024-03-04 13:03:26,203 INFO    MainThread:22592 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-04 13:03:26,203 INFO    MainThread:22592 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-04 13:03:26,206 INFO    MainThread:22592 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-04 13:03:26,211 INFO    MainThread:22592 [wandb_init.py:init():847] run started, returning control to user process
2024-03-04 13:03:27,945 INFO    MainThread:22592 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-04 13:03:27,945 INFO    MainThread:22592 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-04 13:04:49,463 INFO    MainThread:22592 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-04 13:04:49,467 INFO    MainThread:22592 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-04 13:04:49,468 INFO    MainThread:22592 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-04 13:04:51,171 INFO    MainThread:22592 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-04 13:04:51,279 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-04 13:04:51,283 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Configure stats pid to 22592
2024-03-04 13:04:51,284 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-04 13:04:51,284 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2\wandb\settings
2024-03-04 13:04:51,285 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-04 13:04:51,285 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-04 13:04:51,285 INFO    MainThread:22592 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-04 13:04:51,288 INFO    MainThread:22592 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2\wandb\run-20240304_130451-ig468uen\logs\debug.log
2024-03-04 13:04:51,290 INFO    MainThread:22592 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2\wandb\run-20240304_130451-ig468uen\logs\debug-internal.log
2024-03-04 13:04:51,290 INFO    MainThread:22592 [wandb_init.py:init():566] calling init triggers
2024-03-04 13:04:51,291 INFO    MainThread:22592 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-04 13:04:51,291 INFO    MainThread:22592 [wandb_init.py:init():591] re-initializing run, found existing run on stack: dovv3h1w
2024-03-04 13:04:51,293 INFO    MainThread:22592 [wandb_run.py:_finish():1970] finishing run yanzhuo/attention_classifier/dovv3h1w
2024-03-04 13:04:51,293 INFO    MainThread:22592 [jupyter.py:save_history():445] not saving jupyter history
2024-03-04 13:04:51,294 INFO    MainThread:22592 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-04 13:04:51,294 INFO    MainThread:22592 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-04 13:04:51,294 INFO    MainThread:22592 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-04 13:04:51,298 INFO    MainThread:22592 [wandb_run.py:_restore():2193] restore
2024-03-04 13:04:51,299 INFO    MainThread:22592 [wandb_run.py:_restore():2199] restore done
2024-03-04 13:04:55,110 INFO    MainThread:22592 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-04 13:04:55,111 INFO    MainThread:22592 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-04 13:04:55,141 INFO    MainThread:22592 [wandb_run.py:_footer_sync_info():3825] logging synced files
